drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/update_referral_posting_job.rb

```diff
@@ -52,15 +66,14 @@ def update_referral_postings(posting_attrs)
       # build lookup tables for entities referenced in postings; avoid n+1 queries
       postings_by_identifier = posting_attrs
         .map { |h| h.fetch('posting_id') }
-        .compact
+        .compact_blank
```

Comment:
I had no idea! This is amazing! https://apidock.com/rails/Enumerable/compact_blank